BooruSharp.UnitTests/BooruTests.cs

```diff
@@ -489,6 +489,7 @@ public async Task GetRandomsFailAsync(Type t)
         [InlineData(typeof(E926))]
         //[InlineData(typeof(Furrybooru))]
         [InlineData(typeof(Gelbooru))]
+        [InlineData(typeof(Gelbooru), "'o'ne")]
```

Comment:
What does the single quotes on the o mean?

Reply:
The quotes are a part of the the tag as you can see [here](https://gelbooru.com/index.php?page=post&s=list&tags=%27o%27ne). Gelbooru returns the tag encoded for some reason so I fixed it.